spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+  before do
+    request.headers['ACCEPT'] = 'application/json'
+  end
+
+  describe 'POST users#update' do
+    it 'allows user to update the attributes' do
+      user = create(:user)
+      patch :update, params: { id: user.id, user: { name: 'New Name', email: 'newemail@email.com' } }
+      user.reload
+      expect(response).to have_http_status(:ok)
+      expect(user.name).to eql('New Name')
+      expect(user.email).to eql('newemail@email.com')
+    end
+
+    it 'does an error when user update with invalid name' do
+      user = create(:user)
+      patch :update, params: { id: user.id, user: { name: nil } }
+      user.reload
+      expect(response).to have_http_status(:bad_request)
```

Comment:
I'd also add a test to make sure the name is still the same value

Reply:
^ Still needs to be done

Reply:
NVM, I will just add line 34 to the test above, as you suggested.